stubs/Pillow/PIL/Image.pyi

```diff
@@ -204,13 +245,13 @@ class Image:
     def split(self) -> tuple[Image, ...]: ...
     def getchannel(self, channel: int | str) -> Image: ...
     def tell(self) -> int: ...
-    def thumbnail(self, size: tuple[int, int], resample: _Resample = ..., reducing_gap: float = ...) -> None: ...
+    def thumbnail(self, size: tuple[int, int], resample: Resampling | _Resample = ..., reducing_gap: float = ...) -> None: ...
     def transform(
         self,
         size: _Size,
         method: Literal[0, 1, 2, 3, 4],
```

Comment:
Maybe out of scope for this PR, but this kind of thing will get rid of some more of the mypy_primer hits
```suggestion
        method: Transform | Literal[0, 1, 2, 3, 4],
```

But overall I wouldn't worry too much about the remaining mypy_primer hits. The ones in `_pil_constants` are unavoidable and many of the others were preexisting